### PR TITLE
Clarify some words about simple-phpunit

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Could be worth porting to 3.4 too?

### CODE REVIEW

**Review by wouterj — APPROVED** [review]
Thanks for responding to a code issue by clarifying the documentation!

If there are no merge conflicts/changes in 3.4, I propose to merge in 3.4 directly. Otherwise, 4.3 is fine (current is what most people read I think)

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -19,10 +20,13 @@ It comes with the following features:
...
 
-* Provides a ``ClockMock`` and ``DnsMock`` helper classes for time or network-sensitive tests.
+* Provides a ``ClockMock``, ``DnsMock`` and ``ClassExistsMock`` classes for tests
+  sensitive to time, network or class existence.
 
-* Provides a modified version of PHPUnit that does not embed ``symfony/yaml`` nor
-  ``prophecy`` to prevent any conflicts with these dependencies.
+* Provides a modified version of PHPUnit that allows 1. separating the
+  dependencies of your app from those of phpunit to prevent any unwanted
+  constraints to apply; 2. running tests in parallel when a test suite is split
```
Comment (dbrumann) [inline components/phpunit_bridge.rst]: Maybe this, parallelizing tests, could be highlighted more? Do you think it's worth adding a small section to this page giving a brief introduction why it's useful and how it works? If so, I could make a PR when I have some time.

**Review by dbrumann — APPROVED** [review]

File: components/phpunit_bridge.rst (hunk 1 above)
Comment (theofidry) [reply components/phpunit_bridge.rst]: It would be nice indeed, I was totally unaware of that one...

**Comment by javiereguiluz** [comment]
Thanks Nicolas! Merged in 4.2.

File: components/phpunit_bridge.rst (hunk 1 above)
Comment (javiereguiluz) [reply components/phpunit_bridge.rst]: @dbrumann this has been merged, but it'd be nice to add a short section about parallelizing tests. Thanks.

Comment (dbrumann) [reply components/phpunit_bridge.rst]: Yes, I will do an additional PR maybe this weekend.

**Pull request merged**